src/components/dashboard/notifications-tab.less

```diff
@@ -0,0 +1,56 @@
+@notification-item-label-width: 160px;
+@notification-item-content-inset-inline-start: @notification-item-label-width + 20px;
+
+.NotificationsTab-info {
+  .m-info-card;
+}
+
+.NotificationsTab-timeline {
+  .ant-timeline-item-last {
+    .ant-timeline-item-tail {
+      display: none;
+    }
+  }
+
+  .ant-timeline-item-label {
+    width: @activities-item-label-width !important;
+  }
+
+  .ant-timeline-item-tail,
+  .ant-timeline-item-head {
+    inset-inline-start: @activities-item-content-inset-inline-start !important;
+  }
+
+  .ant-timeline-item-content {
+    inset-inline-start: @activities-item-content-inset-inline-start !important;
+    width: 80% !important;
+  }
+
+  @media @edu-media-phone {
+    .ant-timeline-item-label {
+      display: none;
+    }
+
+    .ant-timeline-item-tail,
+    .ant-timeline-item-head,
+    .ant-timeline-item-content {
+      inset-inline-start: 0px !important;
+    }
+  }
+}
+
+.NotificationsTab-notification {
+  .m-overflow-ellipsis;
+}
+
+.NotificationsTab-notificationLabel {
+  font-style: italic;
+}
+
+.NotificationsTab-notificationDescription {
+  margin-right: 5px;
+}
+
+.NotificationsTab-notificationLink {
+  .m-link;
+}
```

Comment:
I'm dying to see how this looks with data!

Reply:
Great find! It was a bug in the client data mapping, thanks! :+1: